fastlane_core/lib/fastlane_core/ui/implementations/shell.rb

```diff
@@ -121,3 +121,9 @@ def verify_interactive!(message)
     end
   end
 end
+
+class String
```

Comment:
Is it a good idea to define String here? If somebody doesn't use `shell` as output (in the future), this won't work. As we now call .deprecated across the code base. Why do we use `puts "str".deprecated` in the code base instead of using `UI.deprecated`?